lib/grppcfp.gi

```diff
@@ -69,6 +69,7 @@ InstallGlobalFunction( IsomorphismFpGroupByPcgs, function( pcgs, str )
         od;
     od;
     H := F / rels;
+    SetIsFinite(H,true);
```

Comment:
I find it rather awkward to add all this `SetIsFinite` calls before each `SetSize`; and it is easy to forget, too. I'd rather add this into a custome setter for `Size`. I can submit a PR doing that tomorrow. Such a setter could also set IsTrivial, IsEmpty, IsNontrivial in one go.

Also, I assume this is only beneficial because setting `IsFinite` does not trigger any/fewer immediate methods than setting `Size`, correct?

Reply:
Yes to both. The setter also is on my to-do list.

Reply:
Also, the IsFinite might be overkill -- I was trying to get rid of calls to immediate methods.